CMakeLists.txt

```diff
@@ -482,9 +482,6 @@ if(SENTRY_BACKEND_CRASHPAD)
 		endif()
 	endif()
 	add_dependencies(sentry crashpad::handler)
-	if(CRASHPAD_WER_ENABLED)
-		add_dependencies(sentry crashpad::wer)
-	endif()
```

Comment:
I’m wondering if this is implied by `crashpad::handler`, or why is it okay to leave this off?

Reply:
I'm thinking that we don't really depend on this since we only really have a runtime dependency that the crashpad_wer.dll exists, and from what I could gather it gets installed fine without this, but it might also be something specific to the sentry-crashpad conan package, that's why I wrote above I'm a bit unsure about this. We shouldn't link to it or use any of its headers right?

Reply:
Hi @Cyriuz, it makes sense to `add_dependencies` on `crashpad::wer` because it makes the dependency explicit (i.e., "make sure to build the `WER` module if you want to build `sentry.dll` with `WER` enabled") precisely because we don't link the target but only require it at runtime (this is especially useful when doing partial rebuilds during development). 

But you are right that adding it as a dependency is unnecessary when using a system crashpad. Can you add it to the build script, but only in the `false` branch of `if(SENTRY_CRASHPAD_SYSTEM)`?

Reply:
That makes sense! I updated as suggested.